mediator/esmFlds.F90

```diff
@@ -27,11 +28,12 @@ module esmflds
          'ice ',&
          'rof ',&
          'wav ',&
-         'glc '/)
+         'glc1',&
```

Comment:
Is this a limitation that only allows up to 2 ice sheets?  If so it seems like a note to that effect should be added here.